docs/user/glossary.rst

```diff
@@ -44,13 +53,10 @@ Not translatable terms
 
 .. versionadded:: 4.5
 
-Glossary terms which are read-only are not meant to be translated. You can use
-this for names or other terms which should not change while translating. Such
-terms are visually highlighted in the glossary sidebar.
-
-The terms can be flagged using :guilabel:`Tools` ↓
-:guilabel:`Mark as read-only`. In the background this toggles the ``read-only``
-flag of the string.
+Flagging certain glossary term translations ``read-only`` by bulk-editing, typing in the flag, or
+by using :guilabel:`Tools` ↓:guilabel:`Mark as read-only` means they can not
```

Comment:
This doesn't render well, see https://github.com/WeblateOrg/weblate/discussions/5857. Fixed in 0fa425e2f94aec7aebfdc4234e01d584309a8607

Reply:
Thank you!